tests/integration/run-javascript.sh

```diff
@@ -23,7 +23,6 @@ if [ ! -d "$SCRIPT_DIR" ]; then
   npm init -y
   npm install
   npm install "$CLIENT_DIR"
-  npm run build
```

Comment:
I think this is not necessary? I was getting an error that `build` does not exist.

Reply:
Hmm. There's a `build` script in this repo's package.json, which is what it's trying to do here, but maybe it's not needed for Jest because it handles compiling TypeScript at runtime.

Reply:
But this runs in the `/tmp/js-test` directory, which has an empty package.json. The `build` for the client does run, but that is above in line 16.